src/components/InputCheckbox/InputCheckbox.stories.tsx

```diff
@@ -176,3 +182,14 @@ export const GroupWithHelpAndError = ({ ...args }) => {
   );
 };
 GroupWithHelpAndError.storyName = 'Group with help and error';
+
+export const GroupDisabled = ({ ...args }) => {
+  return (
+    <Fieldset legend="Legend">
```

Comment:
let's test that putting the disabled attr on the individual inputs vs the fieldset element has the same outcome

Reply:
Tested! Works on individual checkboxes and fieldset.